pkg/limitador/k8s_objects.go

```diff
@@ -64,9 +64,9 @@ func LimitadorDeployment(limitador *limitadorv1alpha1.Limitador) *appsv1.Deploym
 		replicas = int32(*limitador.Spec.Replicas)
 	}
 
-	version := DefaultVersion
+	image := GetLimitadorImageVersion()
```

Comment:
Maybe you are thinking on having extra capabilities for the image and repository, but if not we could simply use `FetchEnv` here like:

```go
image := helpers.FetchEnv(RELATED_IMAGE_LIMITADOR, fmt.Sprintf("%s:%s", LimitadorRepository, DefaultVersion) )
```

Reply:
I only included the logic in a method to be testable in image_test.go